documentation/planning.md

```diff
@@ -71,37 +86,38 @@ As explained above, TesLight supports up to **8 different zones**.
-
-| Zone              | LEDs | Type                      | Cable Length | Physical Channel |
-| ----------------- | ---- | ------------------------- | ------------ | ---------------- |
-| Dash              | 72   | LED strip                 | 100 cm        | 1                |
-| Center Console    | 4    | Fiber cable and injectors | 150 cm        | 2                |
-| Front Doors       | 130  | Light bar (2 in parallel) | 215 cm + 50 cm | 3, 4             |
-| Rear Doors        | 92   | Light bar                 | 405 cm + 45 cm | 5, 6             |
-| Footwell lighting | 4    | LED strip or single LEDs  | 100 cm        | 7, 8             |
-
-So for this build we have a total of around 784 LEDs and need ~20 m cable with 3 wires.
-Also, ~2.5 m of fibre cable is recommended for use around the centre console.
-Make sure to measure the required lengths carefully for individual build.
-
-### Recommended length of light bars
-
-If you are using the light bars, these are the recommended length of the light bars:
-| Location | length of light bars |
-| ------------- | ------------- |
-| Front Doors top (MY 2019) | 100 cm, and shorten about 10 cm  |
-|  Rear Doors top | 75 cm, and shorten a bit  |
-| Front and Rear Doors around handle | 100 cm  |
-| Dashboard | 130 cm, connect 2* 75 cm in series and shorten a bit  |
-| Center console MY2019 (with cover above mobile phone holder) | 2* 85 cm (100 cm, and shorten about 15 cm) |
-| Center console MY2021 | Fibre Wire, tbd light bar |
-
-Make sure to measure the required lengths carefully for individual build.
+#### Recommended Zone Configuration (Tesla Model 3 MY2019)
+| Zone              | LEDs | Luminant Type                     | Physical Channel |
+| ----------------- | ---- | --------------------------------- | ---------------- |
+| Dashboard         | 72   | LED strip or light bar (167 LEDs) | 1                |
+| Center Console    | 4    | Fiber tube and injectors          | 2                |
+| Front Doors       | 130* | Light bar (2 in parallel)         | 3, 4             |
+| Rear Doors        | 130* | Light bar (2 in parallel)         | 5, 6             |
+| Footwells         | 4    | LED strip or single LEDs          | 7, 8             |
+
+*The total count of actual LEDs in this zone is higher. However because two light bars are wired in parallel only the higher LED count of the two light bars matters for the configuration.
+
+Also keep in mind that the actual count of LEDs may differ for your setup as it depends on the exact length of LED strip/Light bar and its density. Make sure to count exact the number of LEDs for your individual build.
+
+### Recommended length of luminants and cables
+
+These are the recommended length of the luminants (LED strip, Fiber tube or LED bar) and cables per installation location. The following table assumes that the TesLight controller is placed below the center console for easy access and wiring.
```

Comment:
2 different tables for same purpose. As a user, I would like to have all information in one place --> one table as before. Even matching zone to location would be more obvious when re-merged in one table.